src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/ciagmigration/ApplicationStartupListener.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.ciagmigration
```

Comment:
All the classes I've written (and I guess what we're proposing) is below this package `ciagmigration` so that it's easy to delete afterwards